From 578a1f2d3937395f3d19cade70050898fd8d2434 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Tue, 3 Feb 2015 15:37:43 -0600
Subject: [PATCH 05/13] remoteproc/pruss: fix the device name for PRU1 on
 AM33xx

The device name used in the match data for the PRU1 core
on AM33xx SoCs has been added incorrectly in the patch,
3b0f8768 ("remoteproc/pruss: use match data instead of
platform data for PRUs"). Fix this typo to reflect the
device name properly, so that the PRU1 can be loaded and
booted successfully. Without this fix, the PRU1 probe
fails with the following trace,
"pru-rproc 4a338000.pru1: missing or incomplete PRU-private data"

Reported-by: Alejandro Hernandez <ajhernandez@ti.com>
Fixes: 3b0f8768 ("remoteproc/pruss: use match data instead of platform data for PRUs")
Signed-off-by: Suman Anna <s-anna@ti.com>
---
 drivers/remoteproc/pruss_remoteproc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/remoteproc/pruss_remoteproc.c b/drivers/remoteproc/pruss_remoteproc.c
index d498ee3..a865295 100644
--- a/drivers/remoteproc/pruss_remoteproc.c
+++ b/drivers/remoteproc/pruss_remoteproc.c
@@ -1228,7 +1228,7 @@ static struct pru_match_private_data pru_match_data[] = {
 		.priv_data	= &pru0_rproc_pdata,
 	},
 	{
-		.device_name	= "4b280000.pru1",
+		.device_name	= "4a338000.pru1",
 		.priv_data	= &pru1_rproc_pdata,
 	},
 	/* AM43xx SoC-specific data */
-- 
1.7.9.5

